staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -230,6 +230,10 @@ func (flags *ApplyFlags) AddFlags(cmd *cobra.Command) {
 
 // ToOptions converts from CLI inputs to runtime inputs
 func (flags *ApplyFlags) ToOptions(cmd *cobra.Command, baseName string, args []string) (*ApplyOptions, error) {
+	if len(args) != 0 {
+		return nil, cmdutil.UsageErrorf(cmd, "Unexpected args: %v", args)
```

Comment:
Just `return nil, fmt.Errorf("unexpected args: %v", args)` and always lowercase.